src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxDocumentConverterBase.cs

```diff
@@ -295,6 +297,12 @@ public IndexFieldsMapping GetKnownFields()
     void HandleArray(IEnumerable itemsToIndex, IndexField field, JsonOperationContext indexContext, ref IndexEntryWriter entryWriter,
         IWriterScope scope, bool nestedArray = false)
     {
+        Span<byte> array = stackalloc byte[1024 * 1024 * 1024];
```

Comment:
??

Reply:
Leftover, I was looking for failing test